# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/qemu/zoom_to_fit.patch
# Copyright (C) 2020 - 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- qemu-8.1.0/ui/gtk.c.vanilla	2023-09-11 15:30:55.922651442 +0200
+++ qemu-8.1.0/ui/gtk.c	2023-09-11 15:31:25.594649491 +0200
@@ -2125,7 +2125,7 @@
                               QemuConsole *con, int idx,
                               GSList *group, GtkWidget *view_menu)
 {
-    bool zoom_to_fit = false;
+    bool zoom_to_fit = true;
     int i;
 
     vc->label = qemu_console_get_label(con);
